src/stories/index.stories.js

```diff
@@ -10,9 +11,11 @@ import ExternalLink from 'js/modules/ExternalLink';
 
 storiesOf('modules', module)
   .add('ExternalLink', () => (
-    <ExternalLink to="http://google.com" iconSize={24}>
-      Go to Google.com
-    </ExternalLink>
+    <IntlProvider locale="en">
```

Comment:
wrapping each story in an `<IntlProvider>` is a bit messy but it was a quick solution. A more holistic solution might be something like this: https://github.com/truffls/storybook-addon-intl

But I had trouble getting the config working as advertised. cc/ @ifsimicoded

Reply:
Let's pair on this post user testing on wednesday